views/movies/edit-movie.hbs

```diff
@@ -0,0 +1,20 @@
+<h2>Edit movie</h2>
+
+<form action="/movies/{{movie._id}}/edit" method="post">
+    <label for="title">Title</label>
+    <input type="text" name="title" id="title" value="{{movie.title}}">
+
+    <label for="genre">Genre</label>
+    <input type="text" name="genre" id="genre" value="{{movie.genre}}">
+
+    <label for="plot">Plot</label>
+    <input type="text" name="plot" id="plot" value="{{movie.plot}}">
+
+    <label for="cast">Cast</label>
+        <select multiple name="cast" id="cast">
+        {{#each celebrities}}
+            <option value="{{_id}}">{{name}}</option>
+        {{/each}}
+            </select>
+
```

Comment:
You forgot a button to submit your form.
`<button type="submit">Edit {{movie.title}}</button>`